src/runtime/hexagon_dma.cpp

```diff
@@ -442,25 +465,34 @@ WEAK int halide_hexagon_dma_buffer_copy(void *user_context, struct halide_buffer
 }
 
 WEAK int halide_hexagon_dma_copy_to_device(void *user_context, halide_buffer_t *buf) {
-    int err = halide_hexagon_dma_device_malloc(user_context, buf);
-    if (err) {
-        return err;
-    }
+    debug(user_context)
+        << "Hexagon: halide_hexagon_dma_copy_to_device (user_context: " << user_context
+        << ", halide_buffer_t: " << buf << ")\n";
```

Comment:
It would be a bit more logical (and concise) to have the label of the parameter in the debug output be `buf` rather than `halide_buffer_t`. Many instances of this in this PR.